db/model/Variable.ts

```diff
@@ -127,6 +127,23 @@ export function parseVariableRows(
     return parsedRows
 }
 
+export async function getMergedGrapherConfigForVariable(
+    variableId: number
+): Promise<GrapherInterface | undefined> {
+    const row = await db.mysqlFirst(
+        `SELECT grapherConfigAdmin, grapherConfigETL FROM variables WHERE id = ?`,
+        [variableId]
+    )
+    if (!row) return
+    const grapherConfigAdmin = row.grapherConfigAdmin
```

Comment:
Could also have just used `&&`'s here 🙂

Reply:
Yeah but I kinda like this style a bit more